src/borg/testsuite/archiver/transfer_cmd.py

```diff
@@ -249,7 +255,8 @@ def local_to_utc(local_naive, tzoffset, tzinfo):
                         assert item.group in ("root", "wheel")
                         assert "hlid" not in item
                     elif item.path.endswith("bdev_12_34"):
-                        assert stat.S_ISBLK(item.mode)
+                        if not is_win32:
+                            assert stat.S_ISBLK(item.mode)
```

Comment:
just wondering: the bdev needed fixing for windows, but the cdev (line 248) didn't?

Reply:
https://github.com/borgbackup/borg/pull/7171/commits/cf61e7d58ae3b3e2fcdc7d9261a23833ac6da7a6

Reply:
ah, ok.